.github/workflows/firebase-hosting-merge-beta.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy to Firebase Hosting on merge to beta
+on:
+  push:
+    branches:
+      - beta
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    environment: beta
+    env:
+      VITE_FIREBASE_API_KEY: ${{secrets.FIREBASE_API_KEY}}
+      VITE_FIREBASE_AUTH_DOMAIN: ${{secrets.FIREBASE_AUTH_DOMAIN}}
+      VITE_FIREBASE_DATABASE_URL: ${{secrets.FIREBASE_DATABASE_URL}}
+      VITE_FIREBASE_PROJECT_ID: ${{secrets.FIREBASE_PROJECT_ID}}
+      VITE_FIREBASE_STORAGE_BUCKET: ${{secrets.FIREBASE_STORAGE_BUCKET}}
+      VITE_FIREBASE_MESSAGING_SENDER_ID: ${{secrets.FIREBASE_MESSAGING_SENDER_ID}}
+      VITE_FIREBASE_APP_ID: ${{secrets.FIREBASE_APP_ID}}
+      VITE_FIREBASE_MEASUREMENT_ID: ${{secrets.FIREBASE_MEASUREMENT_ID}}
+      VITE_MAPBOX_ACCESS_TOKEN: ${{secrets.MAPBOX_ACCESS_TOKEN}}
+      VITE_THUNDERFOREST_API_KEY: ${{secrets.THUNDERFOREST_API_KEY}}
+      VITE_DIRECT_TRAIN_API_URLS: ${{secrets.DIRECT_TRAIN_API_URLS}}
+      VITE_DB_TRANSPORT_REST_API_URLS: ${{secrets.DB_TRANSPORT_REST_API_URLS}}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Dependencies
+        run: yarn
+      - name: Build project
+        run: yarn run build:prod
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          firebaseServiceAccount: "${{ secrets.FIREBASE_SERVICE_ACCOUNT_WTMG_PRODUCTION }}"
+          channelId: live
+          projectId: wtmg-production
```

Comment:
this

Reply:
We should add a target, indeed:
- GH action docs: https://github.com/FirebaseExtended/action-hosting-deploy#target-string
- Firebase docs: https://firebase.google.com/docs/hosting/multisites?authuser=0&hl=en#set_up_deploy_targets

I think we should proceed as follows:
1. Follow the Firebase docs to apply a target name to the prod and beta sites (`firebase target:apply hosting ... ...` - then the target name should be registered.
2. Use those target names in the actions according to the GH action docs.

Reply:
Yes I like this idea